Dockerfile

```diff
@@ -32,23 +55,18 @@ RUN curl -fsSLO "$RCON_URL" \
     && tar xfz rcon-0.10.3-amd64_linux.tar.gz \
     && chmod +x "rcon-0.10.3-amd64_linux/$RCON_BINARY" \
     && mv "rcon-0.10.3-amd64_linux/$RCON_BINARY" "/usr/local/bin/${RCON_BINARY}" \
-    && ln -s "/usr/local/bin/${RCON_BINARY}" /usr/local/bin/rconcli \
     && rm -Rf rcon-0.10.3-amd64_linux rcon-0.10.3-amd64_linux.tar.gz
 
-COPY --chown=steam:steam --chmod=755 backupmanager.sh servermanager.sh includes/* /
-
-EXPOSE 8211/udp
-EXPOSE 25575/tcp
-
-# Workaround for 'Named Volumes'
-RUN mkdir /palworld \
-    && chown steam:steam /palworld
-
-VOLUME [ "/palworld" ]
+COPY --chown=steam:steam --chmod=755 scripts/ /scripts
+COPY --chown=steam:steam --chmod=755 includes/ /includes
+COPY --chown=steam:steam --chmod=755 configs/ /configs
```

Comment:
COPY --chown=steam:steam --chmod=755 configs/rcon.yaml /home/steam/steamcmd/rcon.yaml

We dont need /configs really?
Also that dir has only 1 file, its less descriptive as just pointing to one file